sphinx/source/docs/user_guide/examples/extensions_putting_together_ts.py

```diff
@@ -69,19 +69,21 @@
     // If there is an associated view, this is typically boilerplate.
     this.prototype.default_view = CustomView
 
-    // The define block adds corresponding "properties" to the JS model. These
-    // should normally line up 1-1 with the Python model class. Most property
-    // types have counterparts, e.g. bokeh.core.properties.String will be
-    // ``p.String`` in the JS implementation. Any time the JS type system is not
-    // yet as complete, you can use ``p.Any`` as a "wildcard" property type.
+    // The this.define() block adds corresponding "properties" to the JS
+    // model. These should normally line up 1-1 with the Python model
+    // class. Most property types have counterparts. For example,
+    // bokeh.core.properties.String will correspond to ``String`` in the
+    // JS implementation. Where JS lacks a given type, you can use
+    // ``p.Any`` as a "wildcard" property type.
```

Comment:
This is explanation is out of date. `p.Any` (and similar) are deprecated, and one should use `Any` kind instead, e.g.:
```ts
this.define<Custom.Props>(({Any, String, Ref}) => ({
  text:   [ String ],
  slider: [ Ref(Slider) ],
  whatever: [ Any ],
}))
```
This also means that there is no direct correspondence between bokeh's `String` and bokehjs `String` (kind). The closest is bokeh's `String` and bokehjs' `p.PrimitiveProperty<String>(..., kinds.String, ...)` (I'm using namespace `kinds` to signify the difference between `String` kind and `String` JS built-in type.

Reply:
Would the following be more appropriate, then?

    // ... Most property types have counterparts. For example,
    // bokeh.core.properties.String corresponds to
    // p.PrimitiveProperty<String>(..., kinds.String, ...) in the JS
    // implementation, where ``kinds`` is a custom namespace.
    // If JS lacks a given type, you can use ``Any`` as a "wildcard"
    // property type.

Let me know if this requires a more thorough rewrite, @mattpap.